apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -130,23 +137,81 @@ export default function WpcomBlockEditorNavSidebar() {
 
 interface NavItemProps {
 	item: Post;
+	selected: boolean;
+	statusLabel?: string;
 }
 
-function NavItem( { item }: NavItemProps ) {
+function NavItem( { item, selected, statusLabel }: NavItemProps ) {
+	const buttonClasses = classNames( 'wpcom-block-editor-nav-sidebar__item-button', {
+		'is-selected': selected,
+	} );
+
+	const titleClasses = classNames( 'wpcom-block-editor-nav-sidebar__title', {
+		'is-untitled': ! item.title?.raw,
+	} );
+
 	return (
 		<li>
-			<div>{ item.title.rendered }</div>
-			<pre>{ `/${ item.slug }/` }</pre>
+			<Button className={ buttonClasses }>
+				<div className="wpcom-block-editor-nav-sidebar__title-container">
+					<div className={ titleClasses }>
+						{ item.title?.raw || __( 'Untitled', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'Untitled', { context: 'post title for posts with no title'} )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/jetpack-self-target-warning)